app/assets/javascripts/discourse/lib/utilities.js.es6

```diff
@@ -660,5 +660,32 @@ export function postRNWebviewMessage(prop, value) {
   }
 }
 
+function reportToLogster(name, error) {
+  const data = {
+    message: `${name} theme/component is throwing errors`,
+    stacktrace: error.stack
+  };
+
+  Ember.$.ajax("/logs" + "/report_js_error", {
+    data,
+    type: "POST",
+    cache: false
+  });
+}
+// this function is used in lib/theme_javascript_compiler.rb
+export function rescueThemeError(name, error, api) {
+  console.error(`"${name}" error:`, error);
+  reportToLogster(name, error);
+  const currentUser = api.getCurrentUser();
+  if (!currentUser || !currentUser.get("admin")) {
+    return;
+  }
+  const message = I18n.t("themes.broken_theme_alert", {
+    theme: name
+  });
+  const html = `<div class="broken-theme-alert"><b>${message}</b></div>`;
+  Ember.$("body").prepend(html);
+}
```

Comment:
This could be replaced with vanilla js:

```
const alertDiv = document.createElement("div");
alertDiv.classList.add("broken-theme-alert")
alertDiv.innerText = "message"
document.body.prepend(alertDiv)
```

Note I removed the `<b>` because it doesn't have much interest to me. You should probably set font-weight on `.broken-theme-alert`